llvm/test/Transforms/InstCombine/extract-select-agg.ll

```diff
@@ -56,14 +56,9 @@ define void @test_select_agg_multiuse(i1 %cond, i64 %v1, i64 %v2, i64 %v3, i64 %
 ; CHECK-LABEL: define void @test_select_agg_multiuse(
```

Comment:
Could you also add the tests from https://github.com/llvm/llvm-project/pull/115969?
I've tested them with this patch and it works correctly.

Reply:
I committed the extra tests in https://github.com/llvm/llvm-project/commit/33694245cba0e935f516edf9d20c9062692b1289 and rebased this PR.